### PR TITLE
#449 Hover over the activities in agenda shows hand pointer

### LINKED ISSUE #449: Hover over the activities in the calendar shows a text cursor
## Expected behaviour

Hovering over the activities in the agenda view changes the mouse cursor to a hand pointer instead of text cursor.

## Actual behaviour

Currently, we see a text cursor over the activities in the agenda view

## Steps to reproduce

Have to investigate the styling effect which we are using 
## Checklist

- [x] This issue is linked to the appropriate project.
- [x] This issue has appropriate labels.

### PR DESCRIPTION
**Pull request by Prabhj33t**
## Proposed changes

Resolved  #449 issue with text cursor showing up while hovering on the activities in the agenda. The cursor changes to a hand pointer now.

## Checklist

- [x] Are the issues being addressed linked to this PR?
- [x] Do all commit messages start with the issue number?
- [x] Are all code changes sufficiently tested?
- [x] Are there screenshots for UI changes?

### CODE REVIEW

**Review by daveytech — CHANGES_REQUESTED** [review]
Great work on the ticket Prabh! Could you please include a screenshot for the PR before we accept?

It looks good otherwise!

**Review by wendyyng — APPROVED** [review]
I git checkout to the branch and it worked as expected - hovering over the events in agenda view shows a hand pointer. I also ran yarn delint and it didn't change any formatting.

**Comment by Prabhj33t** [comment]
Here is the screenshot.
MEScree

**Review by daveytech — APPROVED** [review]
Awesome work! Thank you for including the screenshot. This is good to go!

File: webapp/src/index.css [hunk 1]
```diff
@@ -12,6 +12,10 @@ code {
     monospace;
 }
 
+.rbc-agenda-event-cell {
+  cursor: pointer;
```
Comment (camianderson) [inline webapp/src/index.css:16]: Nice work!

**Review by camianderson — APPROVED** [review]
Approved to merge!

**Review by rrabinovitch — APPROVED** [review]
The changes work as expected on my local machine.
My only note is that the PR title doesn't include the relevant issue number, so we might want to edit the title before merging.

**Pull request merged**